python/mxnet/contrib/onnx/mx2onnx/export_model.py

```diff
@@ -28,7 +28,7 @@
 from ._export_helper import load_module
 
 
-def export_model(sym, params, input_shape, input_type=np.float32,
```

Comment:
Instead of changing input_type to a required parameter (and requiring scripts that use this function to be changed), could we check if input_type is a string/dtype value and convert that to a single-item list (for when there's only 1 input)? This would preserve existing behavior, but also allow for passing in a list of input dtypes. WDYT?

Reply:
It definitely make sense to preserve the existing behavior. My concern is that, just like `input_shape`, the `input_type` should also be explicitly passed (they need to be consistent with `input_shape`). The customer may forget to pass it and cause some issues if we provide a default value of it.

Reply:
Up till now, all models have had a float32 input. What if we just set a default like  `input_type=[np.float32]`?

Reply:
Not only the dtype of `input_type`, the number of `input_type` needs to be consistent with `input_shape` as well. Currently we have a bert model test that takes 3 input nodes, and we only passed one `np.float32` as type, which fails the test: https://github.com/apache/incubator-mxnet/blob/9aff832b61fe3132f14d0961bc9ff1798ec670e2/tests/python-pytest/onnx/test_onnxruntime.py#L440

Reply:
I still think having different input types is a special case and not common.. We could automatically set the input_type to preserve backwards compatability and handle all cases and not require script and doc updates like this: 

```
if not isinstance(input_type, list) then:
    input_type = [input_type for _ in range(len(input_shapes))]
```

Reply:
I agree with @josephevans in that we should be compatible with the previous behavior i.e. passing in a single dtype. We can use `isinstance(input_type, list)` to check if more than one dtype is provided

Reply:
Put the default value back. Thank you for the comment!